docs/features/backup-and-restore.rst

```diff
@@ -0,0 +1,67 @@
+===========================
+Backup And Restore Overview
+===========================
+
+Backup and restore are critical components of data management, ensuring that your data is safe and can be recovered in case of loss or corruption. This document provides an overview of the backup and restore process, including best practices, tools, procedures, metrics and more
+
+
+Process Overview
+----------------
+**The Backup process** is managed by ScyllaDB Manager as a whole.
+The overview given here is for a single node, ScyllaDB Manager is responsible to orchestrate backups across the cluster.
+For backup, a snapshot is created, and then the data is copied to a remote location - normally an S3 bucket, Google Cloud Storage, or a similar service.
+
+**The Restore process** is also managed by ScyllaDB manager and it involves copying the data back from the remote location to an empty ScyllaDB node.
+Restoring to a live cluster is not yet supported.
+
+Backup Process
+--------------
+
+#. **Snapshot Creation**: A snapshot of the data is created on the ScyllaDB node.
+   This is a point-in-time copy of the data.
```

Comment:
There's `"sf"` (what a name) parameter to taking snapshot API which controls whether or not to flush a keyspace before snapshotting. If flush is not enabled, it's not point-in-time snapshot. Provided SM requests flush, it's worth mentioning it here.

Reply:
As @Michal-Leszczynski explained: `sf` stands for _skip flush_ - and SM does not issue it. So yes, we flush first, therefore I don't think we need to change anything in this sentence.